--- wradlib-0.8.0.orig/wradlib/util.py	2016-03-10 08:45:53.000000000 -0300
+++ wradlib-0.8.0/wradlib/util.py	2016-07-31 13:11:51.511660379 -0300
@@ -37,7 +37,6 @@
 from scipy import interpolate
 from scipy.ndimage import filters
 from scipy.spatial import cKDTree
-from scipy.stats import nanmean
 from osgeo import ogr
 
 warnings.simplefilter('always', DeprecationWarning)
@@ -354,7 +353,7 @@
             if method == "sum":
                 agg[i] = np.nansum(srcfull)
             elif method == "mean":
-                agg[i] = nanmean(srcfull)
+                agg[i] = np.nanmean(srcfull)
             else:
                 print("Aggregation method not known, yet.")
                 raise Exception()
